release/tar/linux/opensearch-dashboards-tar-build.sh

```diff
@@ -106,23 +106,20 @@ REPO_ROOT=`git rev-parse --show-toplevel`
 ROOT=`dirname $(realpath $0)`; cd $ROOT
 DIR_NAME="opensearch-dashboards-$VERSION"
 WORKING_DIR="$ROOT/$DIR_NAME"
-TARGET_DIR="$ROOT/target"
+TARGET_DIR=`pwd`
```

Comment:
This needs to happen before line 106 because you're changing the current directory there.

Reply:
got your point. Changed it